cheatsheets/gleam-for-python-users.md

```diff
@@ -182,7 +180,10 @@ mul(1, 2)
 
 #### Python
 
-In Python, top level functions are exported by default. There is no notion of private module-level functions.
+In Python, top level functions are exported by default. There is no notion of
+private module-level functions. As a convention functions prefixed with `_` are
+considered private, but there is nothing from preventing you to use those
+outside of the module.
```

Comment:
This expanded prose kinda feels more like "Python for Gleam users" than "Gleam for Python users"

Reply:
That's a fair assessment. Removed the extra information. :sweat_smile: